src/components/Textarea.tsx

```diff
@@ -0,0 +1,161 @@
+import PropTypes, { ValidationMap } from "prop-types";
+import React from "react";
+import { Hint, HintProps } from "./Hint";
+import { ErrorMessage, ErrorMessageProps } from "./ErrorMessage";
+import {
+  FormGroup,
+  FormGroupPropsWithoutChildren,
+  formGroupWithoutChildrenPropTypes
+} from "./FormGroup";
+import { Label, LabelProps } from "./Label";
+import { Attributes, DataAttributes } from "../helpers/Attributes";
+import classNames from "classnames";
+import "lbh-frontend/lbh/components/lbh-textarea/_textarea.scss";
+
+export interface TextareaProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * Id of the textarea.
+   */
+  id: string;
+  /**
+   * Classes to add to the textarea.
+   */
+  className?: string;
+  /**
+   * Name attribute for the textarea.
+   */
+  name: string;
+  /**
+   * Number of rows for the textarea (default: 5).
+   */
+  rows?: number;
+  /**
+   * Value of the textarea.
+   */
+  children?: string;
+  /**
+   * Options for the {@link Label} component.
+   */
+  label: LabelProps;
+  /**
+   * Options for the {@link Hint} component (e.g. text).
+   */
+  hint?: HintProps;
+  /**
+   * Options for the {@link ErrorMessage} component (e.g. text).
+   */
+  errorMessage?: ErrorMessageProps;
+  /**
+   * Options for the {@link FormGroup} wrapper.
+   */
+  formGroup?: FormGroupPropsWithoutChildren;
+  /**
+   * Function to perform when the onChange event is fired.
+   */
+  onChange?(value: string): void;
+  /**
+   * Attribute to
+   * {@link https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html|identify input purpose}
+   * for instance "postal-code" or "username". See
+   * {@link https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill|autofill})
+   * for full list of attributes that can be used.
+   */
+  autocomplete?: string;
```

Comment:
This would be a good case for an enum, given we know the set of valid values. No need to change now, though.